dashboard/config/programming_expressions/applab/Math.abs.json

```diff
@@ -0,0 +1 @@
+{"func":"Math.abs","category":"Math","type":"value","params":["__"],"docFunc":"mathAbs"}
```

Comment:
I'm not sure how easy these are to regenerate, but I think life will generally be easier if we "pretty print" the json here like we do in other config files, so that the diffs are a little easier to read when things change. I'm also not sure how often these will be updated, so feel free to use your judgment here.

Reply:
How do you make something pretty print? It is not ideal to regenerate all of these but I understand the value of what you are saying